sonic-net/sonic-buildimage
# Arista_7060DX5_32 SerDes tunings

#### Why I did it
To ensure strong link signal, avoid flaps and interface linkdowns there should be SerDes tunings for this SKU

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Added a media_settings.json file for the default hwsku and an additional json file for the other two hwsku folders.

#### How to verify it
You can check that the parameters in params in the file are programmed in the Broadcom HSDK and you can view the parameters in APPL_DB.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
